tbconnect/tasks.py

```diff
@@ -0,0 +1,45 @@
+from celery.decorators import periodic_task
+from celery.task.schedules import crontab
+from django.conf import settings
+from temba_client.v2 import TembaClient
+
+from tbconnect.models import TBCheck
+from userprofile.models import HealthCheckUserProfile
+
+
+@periodic_task(run_every=crontab(hour=1))
+def perform_sync_to_rapidpro():
+    if (
+        settings.RAPIDPRO_URL
+        and settings.RAPIDPRO_TB_TOKEN
+        and settings.RAPIDPRO_TBCONNECT_FLOW
+    ):
+        rapidpro = TembaClient(settings.RAPIDPRO_URL, settings.RAPIDPRO_TB_TOKEN)
+
+        for contact in HealthCheckUserProfile.objects.filter(
+            data__follow_up_optin=True
+        ).exclude(data__synced_to_tb_rapidpro=True):
+            check = (
+                TBCheck.objects.filter(msisdn=contact.msisdn)
+                .order_by("-completed_timestamp")
+                .first()
+            )
```

Comment:
We also don't have an index on this. Since each user profile will have multiple checks, this might end up being a very big table.